ipaplatform/base/constants.py

```diff
@@ -23,6 +23,8 @@ class BaseConstantsNamespace:
     NAMED_USER = "named"
     NAMED_GROUP = "named"
     NAMED_DATA_DIR = "data/"
+    NAMED_OPTIONS_VAR = "OPTIONS"
+    NAMED_SSL_ENGINE = None
```

Comment:
The name ``NAMED_SSL_ENGINE`` feels wrong. It's not about SSL but about OpenSSL or cryptographic operations. How about ``NAMED_CRYPTO_ENGINE`` or ``NAMED_OPENSSL_ENGINE``?

Reply:
Sure, renamed `NAMED_SSL_ENGINE` `NAMED_OPENSSL_ENGINE`.